### PR TITLE
Specify wasm features supported by feature_flags

### PR DESCRIPTION
**Pull request by no1wudi**

### CODE REVIEW

File: core/iwasm/aot/aot_loader.c [hunk 1]
```diff
@@ -522,6 +523,56 @@ load_target_info_section(const uint8 *buf, const uint8 *buf_end,
         return false;
     }
 
+    /* Check feature flags */
```
Comment (wenyongh) [inline core/iwasm/aot/aot_loader.c:526]: Had better add a function to check the flags, or the code may be too long here:
```C
static bool
check_feature_flags(uint32 feature_flags);
```

File: core/iwasm/aot/aot_runtime.h [hunk 2]
```diff
@@ -298,6 +312,8 @@ typedef struct AOTTargetInfo {
     uint32 e_version;
     /* Processor-specific flags */
     uint32 e_flags;
+    /* Specify wasm features supported */
+    uint32 feature_flags;
```
Comment (wenyongh) [inline core/iwasm/aot/aot_runtime.h:316]: Not sure whether uint32 is not enough for all features in the feature? Should we use uint64?

File: core/iwasm/aot/aot_loader.c (hunk 1 above)
Comment (no1wudi) [reply core/iwasm/aot/aot_loader.c]: OK

File: core/iwasm/aot/aot_runtime.h (hunk 2 above)
Comment (no1wudi) [reply core/iwasm/aot/aot_runtime.h]: Changed to uint64 now

**Pull request merged**